cli/js/workers_test.ts

```diff
@@ -20,7 +20,10 @@ test(async function workersBasic(): Promise<void> {
 
   tsWorker.onmessage = (e): void => {
     assertEquals(e.data, "Hello World");
-    promise.resolve();
+    // Defer promise.resolve() to allow worker to shut down
+    setTimeout(() => {
+      promise.resolve();
+    });
```

Comment:
Note to self: it should use `worker.terminate()`

Reply:
I think I'm gonna lift workers tests to be integration tests until this issue is resolved (it will look the same as in #4164)